cmd/run.go

```diff
@@ -49,6 +49,11 @@ var (
 	maxRequestBodySize int
 	readBufferSize     int
 	unixDomainSocket   string
+	appHealthEnabled   bool
```

Comment:
```suggestion
	enableAppHealth   bool
```
For consistency with existing CLI flags + Daprd flag `enable-app-health-check`.

Reply:
I'll update tomorrow when i'm on my laptop. Otherwise, if you have maintainer powers and want to commit to my branch, feel free to do so!

Reply:
https://github.com/dapr/cli/pull/1087